File: bidly-backend/src/main/java/com/bidly/bidly_backend/model/Persona.java
package com.bidly.bidly_backend.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "personas")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Persona {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long identificador;

    private String documento;
    private String nombre;
    private String direccion;
    private String estado;
    @Column(name = "foto")
    private byte[] foto;
}

File: bidly-fastapi/app/models/persona.py
from sqlalchemy import Column, Integer, String, LargeBinary
from app.database import Base


class Persona(Base):
    __tablename__ = "personas"

    identificador = Column(Integer, primary_key=True, autoincrement=True)
    documento     = Column(String)
    nombre        = Column(String)
    direccion     = Column(String)
    estado        = Column(String)
    foto          = Column(LargeBinary)
